### ISSUE TITLE
valid-jsdoc with both requireReturnDescription and requireReturnType false still requires one of either

### ISSUE THREAD

**Issue by jurmous** [eslint/eslint#11102]
**Tell us about your environment**

* **ESLint Version:**
5.9.0
* **Node Version:**
v8.11.3
* **npm Version:**
5.6.0
Yarn: 1.12.3
**What parser (default, Babel-ESLint, etc.) are you using?**
"babel-eslint": "^10.0.1"
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    parser: 'babel-eslint',
    rules: {
        'valid-jsdoc': [
            'error',
            {
                prefer: { arg: 'param', argument: 'param', constructor: 'class', return: 'returns', virtual: 'abstract' },
                requireParamDescription: false,
                requireParamType: false,
                requireReturn: false,
                requireReturnDescription: false,
                requireReturnType: false
            },
        ],
    },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
/**
 * @returns
 */
function test() {
    return 'test';
}
```

```bash
 yarn eslint 'index.js' 'src/**/*.js'
```

**What did you expect to happen?**
That ESLint accepts @returns without a description and type

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint fails on an empty @returns. It also fails on a missing @returns statement.

**Are you willing to submit a pull request to fix this bug?**
No.

**Comment by nzakas** [comment]
Thanks for the suggestion. As of yesterday, we have decided to officially [end-of-life JSDoc support](https://eslint.org/blog/2018/11/jsdoc-end-of-life) in ESLint. All JSDoc features are now deprecated and we won't be fixing bugs or making any further improvements to those features.

We are recommending that people transition over to use the [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc) plugin instead of the built-in rules in ESLint.

Thanks for understanding and we apologize for any inconvenience.
